server/manager/src/main/java/org/apache/accumulo/manager/split/SplitTask.java

```diff
@@ -84,8 +84,8 @@ public void run() {
         return;
       }
 
-      var fateInstanceType =
-          FateInstanceType.toType(context.getTableIdToNameMap().get(tablet.getTableId()));
+      var fateInstanceType = FateInstanceType
+          .fromNamespaceOrTableName(context.getTableIdToNameMap().get(tablet.getTableId()));
```

Comment:
Going from a table id to a table name is tricky because there can be race conditions with rename table.  Generally its best to operate only on the table id when its present.

Was thinking we could add something like the following to FateInstanceType to address this case.

```java
  public static FateInstanceType fromTableId(TableId tableId) {
    if(RootTable.ID.equals(tableId) || MetadataTable.ID.equals(tableId) || FateTable.ID.equals(tableId)) {
      return META;
    }
    
    return USER;
  }
```

However one thing I do not like about the above is thatit has the potential to drift in the case  a new "accumulo.X" table is added.

Reply:
It would be outside the scope of this PR, but could address the drift issue by creating a new internal AccumuoTables class and move the RootTable, MetadataTable, and FateTable classes into it.  Then it could he an `Set<TableId> allIds()` method.